frontend/src/component/releases/ReleasePlanTemplate/TemplateForm/MilestoneList/MilestoneCard/MilestoneCardDragHandle.tsx

```diff
@@ -0,0 +1,54 @@
+import DragIndicator from '@mui/icons-material/DragIndicator';
+import { styled } from '@mui/material';
+import { ScreenReaderOnly } from 'component/common/ScreenReaderOnly/ScreenReaderOnly';
+import type { FC } from 'react';
+
+const DragButton = styled('button')(({ theme }) => ({
+    padding: 0,
+    cursor: 'grab',
+    transition: 'background-color 0.2s ease-in-out',
+    backgroundColor: 'inherit',
+    border: 'none',
+    borderRadius: theme.shape.borderRadiusMedium,
+    color: theme.palette.text.secondary,
+    ':hover, :focus-visible': {
+        outline: 'none',
+        '.draggable-hover-indicator': {
+            background: theme.palette.table.headerHover,
+        },
+    },
+}));
+
+const DraggableContent = styled('span')(({ theme }) => ({
+    paddingTop: theme.spacing(2),
+    paddingInline: theme.spacing(0.5),
+    display: 'block',
+    height: '100%',
+    width: '100%',
+}));
+
+const DraggableHoverIndicator = styled('span')(({ theme }) => ({
+    display: 'block',
+    paddingBlock: theme.spacing(0.75),
+    borderRadius: theme.shape.borderRadiusMedium,
+    '> svg': {
+        verticalAlign: 'bottom',
+    },
+}));
+
+type Props = {
+    dragItemRef: React.RefObject<HTMLElement>;
+};
+
+export const MilestoneCardDragHandle: FC<Props> = ({ dragItemRef }) => {
+    return (
+        <DragButton tabIndex={-1} type='button'>
+            <DraggableContent ref={dragItemRef}>
+                <DraggableHoverIndicator className='draggable-hover-indicator'>
+                    <DragIndicator aria-hidden />
+                </DraggableHoverIndicator>
+                <ScreenReaderOnly>Drag to reorder</ScreenReaderOnly>
```

Comment:
The double span here serves a few purposes:
1. It allows you to start the drag anywhere the side of the card (buttons don't work great with drag n drop, so we'll use a span to make up for it)
2. Because the outer span is the height of the card, it also works to calculate whether we can or can not flip the card.
3. The inner span is just there for hover / focus indication.